code/modules/mob/living/bloodcrawl.dm

```diff
@@ -0,0 +1,193 @@
+				sleep(30)
+				GibEat(kidnapped)
+				playsound(get_turf(src),'sound/misc/Demon_consume.ogg', 100, 1)
+				sleep(30)
+				src << "<B>You devour [kidnapped]. Your health is fully restored.</B>"
+				src.adjustBruteLoss(-1000)
+				kidnapped.ghostize()
+				qdel(kidnapped)
+				new /obj/effect/gibspawner/human(get_turf(src))///Somewhere a janitor weeps.
+				if (istype(src, /mob/living/simple_animal/slaughter)) //rason, do not want humans to get this
+
+					var/mob/living/simple_animal/slaughter/demon = src
+
+					demon.devoured++
+					//src.kidnapped = null
+					if (demon.devoured == 5)
+						src.mind.current.verbs += /mob/living/simple_animal/slaughter/proc/goreThrow
+						src.mind.current.verbs += /mob/living/simple_animal/slaughter/proc/bloodSac
+						src << "<span class='notice'> You have consumed enough to be able to summon Excess Gore.</span>"
+			else
+				sleep(6)
+				if (animation)
+					qdel(animation)
+			src.notransform = 0
+
+/mob/living/proc/phasein(var/obj/effect/decal/cleanable/B)
+	if(src.notransform)
+		src << "<B>Finish eating first!</B>"
+	else
+		var/atom/movable/overlay/animation = new /atom/movable/overlay( B.loc )
+		animation.name = "odd blood"
+		animation.density = 0
+		animation.anchored = 1
+		animation.icon = 'icons/mob/mob.dmi'
+		animation.icon_state = "jauntup" //Paradise Port:I reversed the jaunt animation so it looks like its rising up
+		animation.layer = 5
+		animation.master = B.loc
+		animation.dir = src.dir
+		flick("jauntup",animation)
+		src.loc = B.loc
+		src.client.eye = src
+		if (prob(25))
+			var/list/voice = list('sound/hallucinations/behind_you1.ogg','sound/hallucinations/im_here1.ogg','sound/hallucinations/turn_around1.ogg','sound/hallucinations/i_see_you1.ogg')
+			playsound(get_turf(src), pick(voice),50, 1, -1)
+		src.visible_message("<span class='warning'><B>\The [src] rises out of \the [B]!</B>")
+		playsound(get_turf(src), 'sound/misc/exit_blood.ogg', 100, 1, -1)
+		qdel(src.holder)
+		src.holder = null
+		sleep(6)
+		if(animation)
+			qdel(animation)
+
+/obj/effect/decal/cleanable/blood/CtrlClick(mob/living/user)
+	..()
+	if(user.bloodcrawl)
+		if(user.holder)
+			user.phasein(src)
+		else
+			user.phaseout(src)
+
+
+
+/obj/effect/decal/cleanable/trail_holder/CtrlClick(mob/living/user)
+	..()
+	if(user.bloodcrawl)
+		if(user.holder)
+			user.phasein(src)
+		else
+			user.phaseout(src)
+
+
+
+/turf/CtrlClick(var/mob/living/user)
+	..()
+	if(user.bloodcrawl)
+		for(var/obj/effect/decal/cleanable/B in src.contents)
+			if(istype(B, /obj/effect/decal/cleanable/blood) || istype(B, /obj/effect/decal/cleanable/trail_holder))
+				if(user.holder)
+					user.phasein(B)
+					break
+				else
+					user.phaseout(B)
+					break
+
+/obj/effect/dummy/slaughter //Can't use the wizard one, blocked by jaunt/slow
+	name = "odd blood"
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "nothing"
+	var/canmove = 1
+	density = 0
+	anchored = 1
+	//invisibility = INVISIBILITY_OBSERVER
```

Comment:
There was a disagreement with it in a previous comment when i put that in, I commented it out instead of removing it..will do that when i fix other addressed issues.